lib/db_connection/connection.ex

```diff
@@ -109,6 +109,7 @@ defmodule DBConnection.Connection do
 
     s = %{mod: mod, opts: opts, state: nil, client: :closed, broker: broker,
           regulator: regulator, lock: nil, queue: queue, timer: nil,
+          hibernate_timer: nil,
```

Comment:
Please use `idle_timer` as the timer is for idle and not hibernating, but hibernate is a possible side effect of going idle.